task/github-open-pr/0.1/README.md

```diff
@@ -0,0 +1,114 @@
+# GitHub Open Pull Request
+
+GitHub Open Pull Request is part of a collection of GitHub tasks to help working
+with the [GitHub API](https://docs.github.com/en/rest/reference).
+
+## GitHub token
+
+This task expects a secret set in the kubernetes secret `github`
+with a GitHub token in the key `token`; you can easily create it on the
+command line with `kubectl` like this :
+
+```
+kubectl create secret generic github --from-literal token="MY_TOKEN"
+```
+
+## Open Pull Request
+
+The `github-open-pr` task uses the [pull request api](https://docs.github.com/en/rest/reference/pulls#create-a-pull-request)
+to open pull requests on Github. It is able to fill in a title and body of the pull request.
+
+
+### Install the Task
+
+```
+kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/main/task/github-open-pr/0.1/github-open-pr.yaml
+```
+
+### Parameters
+
+* **GITHUB_HOST_URL**: The GitHub host domain _default:_ `api.github.com`
+* **REPO_FULL_NAME**: The GitHub repository full name, _e.g:_ `tektoncd/catalog`
+* **API_PATH_PREFIX**: The GitHub Enterprise has a prefix for the API path. _e.g:_ `/api/v3`
+* **HEAD**: The name of the branch where your changes are implemented. _e.g.: `new-feature-i-created`
+* **BASE**: The name of the branch you want the changes pulled into. _e.g.: `develop`
+* **BODY**: The body description of the pull request. _e.g.: `The following PR includes these changes...`
+* **BASE**: The title of the pull request. _e.g.: `Feature X`
```

Comment:
This seems like a typo, name should be 'TITLE'

Reply:
Thanks! Pushed fix for the typo.